### ISSUE TITLE
Timed Text WG (TTML, WebVTT)

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#177]
The Working Group is updating its charter:
  https://w3c.github.io/charter-timed-text/

Most notably:
- 2D, 3D, 360º, AR and VR
- Live captioning
- profile for audio descriptions

**Comment by samuelweiler** [comment]
This draft removes some language from the "coordination" section of the [current charter](https://www.w3.org/2018/05/timed-text-charter.html) that is relevant (e.g. privacy and security review) and instead asks for "horizontal" review from WGs that we don't typically consider to be horizontal areas.  The latter is just a wording issue; the former is a problem.

Perhaps instead copy text from the [current charter template](https://w3c.github.io/charter-drafts/charter-template.html)?

Charter history is incomplete - it says "none" where I trivially spot some changes (e.g. in staff contact and FTE allocation).  Those canaries suggest there were other changes.  Please fill out the history.

**Comment by nigelmegitt** [comment]
@samuelweiler Horizontal review isn't so clearly defined; The process links to the [guide](https://www.w3.org/Guide/process/charter.html#horizontal-review) which seems to allow for some flexibility. Nevertheless, just changing the text above the list from:

> the Working Group SHALL seek horizontal review with the W3C Horizontal Groups listed below

to

> the Working Group SHALL seek review, including horizontal review, with the W3C Groups listed below

would seem to satisfy your first comment? The text from the current charter template was already rejected by the TTWG.

**Comment by r12a** [comment]
[from Fuqiao]

In the Coordination section, I think the Working Group shall also seek horizontal review from the i18n WG, since a lot of the work in the group are i18n-related, including writing mode, ruby text annotations, and text direction. 

[from me]
1.1 Success Criteria lists a number of goals which includes a bullet point for accessibility.  It's not clear to me why that point doesn't also include internationalization.

Btw, i hope that the group will address i18n recommendations from TTML2 that were deferred. Not sure the charter needs to say that though.

"To promote interoperability with the overall web platform, and for all Recommendation-track documents, the Working Group SHALL seek horizontal review with the W3C Horizontal Groups listed below."

No horizontal groups are listed below other than the TAG. 

Why aren't we using the wording from the latest version of the template?

**Comment by nigelmegitt** [comment]
I agree i18n should be added to the review groups list, this seems to be a mistake. Previously it was listed under liaison and that section was refactored.

**Comment by plehegar** [comment]
As mentioned in #52, I do believe that the charter needs to go the AC with the wording of the charter template. Doing otherwise giving the issues we have with horizontal reviews not getting done would be inappropriate imho.

**Comment by nigelmegitt** [comment]
The trouble is the charter template wording is broken, see my comments on #52. I think we should be able to reach a middle ground, but in particular requiring 3 months notice before every CR publication puts the brakes on WG work too much, and is almost certainly unncessary, and very likely unintended.

**Comment by brewerj** [comment]
Accessibility review is now completed, with both Michael and Judy's reviews. 
Also I restored the Privacy Review Completed tag after accidentally removing it.
